collectors/test-results/jira-xray/src/main/java/com/capitalone/dashboard/client/api/domain/Defect.java

```diff
@@ -0,0 +1,45 @@
+package com.capitalone.dashboard.client.api.domain;
+
+import com.atlassian.jira.rest.client.api.domain.BasicIssue;
+
+import java.net.URI;
+
+/**
+ * This class will get the details defect
+ */
+public class Defect extends BasicIssue {
+    private String summary;
+    private String myStatus;
+    // private com.atlassian.jira.rest.client.api.domain.Status status;
```

Comment:
If this is not needed then remove it. Dead code should not be left in

Reply:
done